src/appleseed.studio/mainwindow/rendering/renderingmanager.cpp

```diff
@@ -31,6 +31,7 @@
 #include "renderingmanager.h"
 
 // appleseed.studio headers.
+#include "mainwindow/droppedmaterialmenu.h"
```

Comment:
Probably not needed anymore.

Reply:
yeah, was just waiting for confirmation before removing completly droppedmaterialmenu